### PR TITLE
cargo request console pings supply radio when someone makes a new request

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: cargo request console pings supply radio when someone makes a new request
/:cl:

its hard to notice new requests if you arent 24/7 near the supply console, better interactivity doesnt hurt

### CODE REVIEW

File: code/modules/cargo/console.dm [hunk 1]
```diff
@@ -22,13 +26,18 @@
...
 	var/obj/item/circuitboard/computer/cargo/board = circuit
 	contraband = board.contraband
 	if (board.obj_flags & EMAGGED)
 		obj_flags |= EMAGGED
 	else
 		obj_flags &= ~EMAGGED
 
+/obj/machinery/computer/cargo/Destroy()
+	. = ..()
+	QDEL_NULL(radio)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/console.dm]: run inher after qdel

**Comment by MCterra10** [comment]
Can we have a toggle button on the requests console so we can disable it if someone starts spamming requests?

**Comment by MCterra10** [comment]
Cargo console would be better because they can just enable it otherwise

**Comment by Tlaltecuhtli** [comment]
it has a 30 seconds cooldown before a new message can appear

**Comment by MCterra10** [comment]
Very nice, very nice

**Pull request merged**